lib/events/api.go

```diff
@@ -145,6 +147,10 @@ type IAuditLog interface {
 	// SearchSessionEvents returns session related events only. This is used to
 	// find completed session.
 	SearchSessionEvents(fromUTC time.Time, toUTC time.Time) ([]EventFields, error)
+
+	// Wait waits for resources to be released and outstanding requests to
+	// complete after calling Close method
+	Wait(context.Context) error
```

Comment:
If `IAuditLog` was not embedded everywhere, `Wait` would be a reasonable name. Unfortunately `IAuditLog` is embedded everywhere, for example `auth.Client` and `auth.AuthWithRoles` where `Wait` is confusing.

It should be named something more specific.